packages/@react-aria/grid/src/useGrid.ts

```diff
@@ -14,28 +14,29 @@ import {AriaLabelingProps, DOMProps, KeyboardDelegate} from '@react-types/shared
 import {filterDOMProps, mergeProps, useId} from '@react-aria/utils';
 import {GridCollection} from '@react-types/grid';
 import {GridKeyboardDelegate} from './GridKeyboardDelegate';
+import {gridKeyboardDelegates} from './utils';
 import {GridState} from '@react-stately/grid';
 import {HTMLAttributes, RefObject, useMemo} from 'react';
 import {useCollator, useLocale} from '@react-aria/i18n';
 import {useSelectableCollection} from '@react-aria/selection';
 
-const gridIds = new WeakMap<GridState<unknown, GridCollection<unknown>>, string>();
-
-interface GridProps extends DOMProps, AriaLabelingProps {
+export interface GridProps extends DOMProps, AriaLabelingProps {
   ref: RefObject<HTMLElement>,
   isVirtualized?: boolean,
-  keyboardDelegate?: KeyboardDelegate
+  keyboardDelegate?: KeyboardDelegate,
+  focusMode?: 'row' | 'cell'
```

Comment:
Also `focusMode` might be confusing, especially since grid and cells take different options? API suggestions welcome.